MarketplaceService/MarketplaceService.csproj

```diff
@@ -4,6 +4,7 @@
         <TargetFramework>netcoreapp3.1</TargetFramework>
         <UserSecretsId>2145df81-9941-4fef-bc85-93ddde474a1f</UserSecretsId>
         <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+        <RootNamespace>marketplaceservice</RootNamespace>
```

Comment:
Verander root namespace naar MarketplaceService

Reply:
Dit geeft naming errorrs dus beter laten we het zo staan. Alternatief is om het hele project te renamen naar Microservice ipv Service

Reply:
Als ik het refactor krijg ik alleen een import issue als ik de service zelf aanroep. 
Dat kan je oplossen door dit bovenin de file te zetten zodat de juiste service wordt geimporteerd;
using MarketplaceService = MarketplaceService.Services.MarketplaceService;